Fixed rendering of Martin Fowler Quote in Building your Application

jcloutz opened the pull request:
This should fix the quote being rendered incorreclty. I also removed all the line breaks int the file.

jcloutz committed:
fixed blockquote and removed linebreaks "Building your Application"

philsturgeon commented:
Why remove the line wrapping? They are added intentionally, to keep things near 120 characters. Markdown will not (or should not) actually parse those, so it doesnt matter. I can't merge this unless you put those line breaks back :-/

markushausammann commented:
Look at the page, because of the linebreaks, the quote is rendered wrongly!! Just merge it, it's a legit request.

jcloutz committed:
Merge branch 'gh-pages' of github.com:codeguy/php-the-right-way into gh-pages

jcloutz committed:
replaced line breaks and added block quote markup for multiple lines of the fowler quote

jcloutz commented:
I replaced the line breaks, and added the proper `>` to each line of the quote. Sorry about going over the 125 char limit. I saw that after i sent the initial pull request but didn't have the chance fix it yet.
